src/System.Collections.NonGeneric/tests/Performance/Perf.HashTable.cs

```diff
@@ -69,14 +78,16 @@ public void Add(Hashtable table)
         {
             foreach (var iteration in Benchmark.Iterations)
             {
+                Hashtable tableCopy = new Hashtable(table);
                 using (iteration.StartMeasurement())
                 {
-                    table.Add("key1", "value"); table.Add("key2", "value"); table.Add("key3", "value");
-                    table.Add("key4", "value"); table.Add("key5", "value"); table.Add("key6", "value");
-                    table.Add("key7", "value"); table.Add("key8", "value"); table.Add("key9", "value");
+                    for (int i = 0; i < 40000; i++)
+                    {
+                        tableCopy.Add(i * 10 + 1, "value"); tableCopy.Add(i * 10 + 2, "value"); tableCopy.Add(i * 10 + 3, "value");
+                        tableCopy.Add(i * 10 + 4, "value"); tableCopy.Add(i * 10 + 5, "value"); tableCopy.Add(i * 10 + 6, "value");
+                        tableCopy.Add(i * 10 + 7, "value"); tableCopy.Add(i * 10 + 8, "value"); tableCopy.Add(i * 10 + 9, "value");
```

Comment:
Yeah.. that's expected. It was either boxed ints, string concatenation, or pre-building 360000 boxed ints/strings in a local array before the measurement starts and then add those within the loops. I may switch to the pre-built method later down the line if results are being skewed too much, but I think it is fine for now.